src/libexpr-c/nix_api_value.cc

```diff
@@ -20,7 +20,7 @@
 #  include "gc_cpp.h"
 #endif
 
-// Helper function to throw an exception if value is null
+// Helper function to throw an exception if value is null or in an invalid state
```

Comment:
This wasn't actually changed.

```suggestion
// Helper function to throw an exception if value is null
```